pfb-analysis/scripts/setup_database.sh

```diff
@@ -7,15 +7,32 @@ NB_POSTGRESQL_USER=gis
 NB_POSTGRESQL_PASSWORD=gis
 
 # Set defaults for overridable configuration params
-PFB_WORK_MEM="${PFB_WORK_MEM:-2048MB}"
 PFB_CHECKPOINT_COMPLETION="${PFB_CHECKPOINT_COMPLETION:-0.8}"
-PFB_MAX_WAL_SIZE="${PFB_MAX_WAL_SIZE:-2GB}"
+# Disable to improve performance, if disabled, data loss occurs on server crash (ok for analysis)
+PFB_FSYNC="${PFB_FSYNC:-off}"
+# Only one process at a time, can be higher, set to 1/4 of available system memory
+PFB_MAINTENANCE_WORK_MEM="${PFB_MAINTENANCE_WORK_MEM:-1024MB}"
+# WAL is not useful for us, so keep small to limit disk usage
+PFB_MAX_WAL_SIZE="${PFB_MAX_WAL_SIZE:-256MB}"
```

Comment:
Yeah, all of the moving pieces here were a bit confusing to me. It does not seem possible to completely turn off WAL, even if fsync=off. The [continuous archiving](https://www.postgresql.org/docs/9.6/static/continuous-archiving.html) docs state that "_At all times_, PostgreSQL maintains a write ahead log (WAL)...".

@hectcastro is this something you're more familiar with? We don't need crash recovery since this is a temp database for each analysis run so it just uses up unnecessary system resources.